code/modules/lighting/light_planes.dm

```diff
@@ -0,0 +1,63 @@
+/obj/abstract/screen/plane
+	name = ""
+	screen_loc = "CENTER"
+	blend_mode = BLEND_MULTIPLY
+	layer = 1
+
+/obj/abstract/screen/plane/New(var/client/C)
+	..()
+	if(istype(C))
+		C.screen += src
+	verbs.Cut()
+
+/obj/abstract/screen/plane/master
+	icon = 'icons/mob/screen1.dmi'
+	appearance_flags = NO_CLIENT_COLOR | PLANE_MASTER | RESET_TRANSFORM | RESET_COLOR | RESET_ALPHA
+	color = LIGHTING_PLANEMASTER_COLOR  // Completely black.
+	plane = LIGHTING_PLANE
+	mouse_opacity = 0
+
+//poor inheritance shitcode
+/obj/abstract/screen/backdrop
```

Comment:
This needs to be `PLANE_MASTER` if you want it to work.

Reply:
I don't follow, what needs to be `PLANE_MASTER`?

Reply:
What needs to be plane_master?